From c060e718d74972ed31a85bdd02e5e3a8e517ee1f Mon Sep 17 00:00:00 2001
From: Xiao Wang <jasowang@redhat.com>
Date: Thu, 21 Jul 2016 06:40:30 +0200
Subject: [PATCH 03/35] pci: Introduce define for PM capability version 1.1

RH-Author: Xiao Wang <jasowang@redhat.com>
Message-id: <1469083246-12219-4-git-send-email-jasowang@redhat.com>
Patchwork-id: 71250
O-Subject: [RHEL7.3 qemu-kvm-rhev PATCH 03/19] pci: Introduce define for PM capability version 1.1
Bugzilla: 1343092
RH-Acked-by: Laszlo Ersek <lersek@redhat.com>
RH-Acked-by: Laurent Vivier <lvivier@redhat.com>
RH-Acked-by: Dmitry Fleytman <dfleytma@redhat.com>

From: Dmitry Fleytman <dmitry.fleytman@ravellosystems.com>

Signed-off-by: Dmitry Fleytman <dmitry.fleytman@ravellosystems.com>
Signed-off-by: Leonid Bloch <leonid.bloch@ravellosystems.com>
Reviewed-by: Michael S. Tsirkin <mst@redhat.com>
Signed-off-by: Jason Wang <jasowang@redhat.com>
(cherry picked from commit 83f17ed278dc5a8b3f7f2c8765f17ed4302b74e8)
Signed-off-by: Miroslav Rezanina <mrezanin@redhat.com>
---
 include/hw/pci/pci_regs.h | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/include/hw/pci/pci_regs.h b/include/hw/pci/pci_regs.h
index ba8cbe9..7a83142 100644
--- a/include/hw/pci/pci_regs.h
+++ b/include/hw/pci/pci_regs.h
@@ -1 +1,3 @@
 #include "standard-headers/linux/pci_regs.h"
+
+#define  PCI_PM_CAP_VER_1_1     0x0002  /* PCI PM spec ver. 1.1 */
-- 
1.8.3.1

